week08_pomdp/practice_pytorch.ipynb

```diff
@@ -6,11 +6,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import os\n",
     "import sys\n",
     "if 'google.colab' in sys.modules:\n",
     "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/0ccb0673965dd650d9b284e1ec90c2bfd82c8a94/week08_pomdp/atari_util.py\n",
     "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/0ccb0673965dd650d9b284e1ec90c2bfd82c8a94/week08_pomdp/env_pool.py\n",
-    "\n",
+    "    # Setup the attari driver for video games\n",
+    "    !wget -q https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/setup_colab.sh -O- | bash\n",
+    "    !touch .setup_complete\n",
+    "    # Install packages dependencies\n",
+    "    os.system(\"pip install -q scipy==1.1.0\")\n",
+    "    os.system(\"pip install gym[atari,accept-rom-license]==0.21.0.\")\n",
```

Comment:
Are you sure a `.` at the end of the version number should be there?

Reply:
Fixed.